hexrd/ui/calibration_crystal_editor.py

```diff
@@ -155,28 +157,15 @@ def position(self, v):
             blocker = QSignalBlocker(w)  # noqa: F841
             w.setValue(v[i])
 
-    @property
-    def matrix_scaling(self):
-        scaling = np.array([np.sqrt(2)] * 9)
-        diagonal_indices = [0, 4, 8]
-        for i in diagonal_indices:
-            scaling[i] = 1
-
-        return scaling
-
     @property
     def inverse_stretch(self):
-        m = (self.stretch_matrix * self.matrix_scaling).reshape(3, 3)
-        m = np.linalg.inv(m).flatten()
-        indices = [0, 4, 8, 5, 2, 1]
-        return [m[i] for i in indices]
+        m = np.array(self.stretch_matrix).reshape(3, 3)
```

Comment:
self.stretch matrix is the raw user input?

Reply:
Yes, `self.stretch_matrix` returns a flattened list of the matrix values from the GUI, with indices `[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]`, where the 3D shape is:
```
0 1 2
3 4 5
6 7 8
```

Reply:
It then gets reshaped by numpy into a 3x3 matrix.